node/src/main/scala/org/bitcoins/node/Node.scala

```diff
@@ -103,26 +235,22 @@ trait Node extends NodeApi with ChainQueryApi with P2PLogger {
                                     CompactFilterDAO())
   }
 
+  //todo think of a better name
+  def handlePeerGossipMessage(message: Any): Unit
```

Comment:
We don't want to be using `Any`, should make a `sealed trait` that encapsulates `AddrMessage` and `AddrV2Message`

Reply:
I think the function name is good though